src/lib.rs

```diff
@@ -145,16 +145,16 @@ pub use self::filters::{
 #[doc(hidden)]
 pub use self::filters::ws::ws;
 #[doc(hidden)]
-pub use self::redirect::redirect;
+pub use self::redirect::{redirect, temporary_redirect};
```

Comment:
I'd just let the more specialized functions exist only in the `redirect` module.